src/main/java/org/chameleoncloud/PostLogoutPageProvider.java

```diff
@@ -39,7 +37,10 @@ public Response get(@QueryParam("client_id") String clientId) {
             }
         }
 
-        return form.createForm("post-logout.ftl");
+        Response r = form.createForm("post-logout.ftl");
+        CacheControl cc = new CacheControl();
+        cc.setNoCache(true);
+        return Response.fromResponse(r).cacheControl(cc).build();
```

Comment:
Seemed easier to reimplemented NoCache annotation like this than add new dependency or a custom decorator in our code.